Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py

```diff
@@ -520,6 +526,7 @@ def main():
         params = demisto.params()
         refresh_token = params.get('refresh_token', '')
         self_deployed = params.get('self_deployed', False)
+        redirect_uri = params.get('redirect_uri', '')
```

Comment:
Are you sure the redirect_uri can have an empty string as default? because I saw in MicrosoftApiModule it gets something.
just to make sure.